src/clip.ts

```diff
@@ -2,7 +2,8 @@ import { isNote, shuffle, expandStr } from './utils';
 import { getChord } from './chord';
 
 /* tslint:disable:no-var-requires */
-const browserClip = typeof window !== 'undefined' && require('./browserClip');
+const browserClip =
+  typeof window !== 'undefined' && require('./browserClip').browserClip;
 
 /**
  * Get defauly params for a clip, such as root note, pattern etc
```

Comment:
This is not part of the PR but just noticed it, since there is another comment on this PR, would you kindly fix this typo "defauly" as part of this PR.

Reply:
Solved in https://github.com/scribbletune/scribbletune/pull/113/commits/1ff91750cb3dad5b79bff216af47fcc83b9f781c